00_сompilation_process/00_00_preprocessor/00_00_02_header_guard/00_00_02_header_A.h

```diff
@@ -0,0 +1,8 @@
+#ifndef MY_HEADER_A
+#define MY_HEADER_A
+
+int A_func() {
+    return 0;
+}
+
+#endif // MY_HEADER_A
```

Comment:
No newline at the end of the file, it is a minor issue, so I think we may ignore it.

Also applicable for some other files.

Reply:
Done